SU-HSDO/suhumsci
# SHS-5495: Update border color on combo based select fields

# READY FOR REVIEW

## Summary
Update border color on select fields in views to match the Search by Keyword textfield

## Need Review By (Date)
02/23

## Urgency
low

## Steps to Test
1. Go to the following pages:
   - https://feminist.suhumsci.loc/news-events/upcoming-events
   - https://creativewriting.suhumsci.loc/books
   - https://hs-colorful.suhumsci.loc/default-views/courses
   - https://hs-traditional.suhumsci.loc/default-views/courses
2. Confirm the border color of the select fields match the border color of the Search by Keyword textfield

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
